lib/util/git/index.spec.ts

```diff
@@ -10,24 +10,27 @@ import {
   UNKNOWN_ERROR,
 } from '../../constants/error-messages';
 import { newlineRegex, regEx } from '../regex';
+import * as _modifiedAuth from './auth';
 import * as _behindBaseCache from './behind-base-branch-cache';
 import * as _conflictsCache from './conflicts-cache';
 import * as _modifiedCache from './modified-cache';
 import type { FileChange } from './types';
-import * as git from '.';
 import { setNoVerify } from '.';
+import * as git from '.';
```

Comment:
please avoid this change, drop the line above this line and prefix that function call with `git.`